.vscode/settings.json

```diff
@@ -16,5 +16,6 @@
         "ostr",
         "Upcall"
     ],
-    "gradle.nestedProjects": true
+    "gradle.nestedProjects": true,
+    "java.format.enabled": false
```

Comment:
Prevent the extension for auto-formatting files.
Spotless will exclusively handle the formatting during build time.